ulib/fs/VS/Makefile

```diff
@@ -11,13 +11,12 @@ ifeq ($(OS),Windows_NT)
 else
   FSC     = fsharpc
   # If can't find msbuild, use xbuild, but throw a warning
-  MSBUILD = $(shell which msbuild || (echo '\n\n\033[0;31mWarning:\033[0m could not find "msbuild", trying (deprecated) "xbuild"\n\n'>&2; which xbuild))
+  MSBUILD = $(shell which msbuild || (echo "dotnet msbuild"))
+  #MSBUILD = dotnet msbuild
```

Comment:
Can you please say more about this change, the earlier code was looking for `xbuild` in case `msbuild` is not found, but now we are just printing the message `dotnet msbuild`, why?

Also what's up with the commented line?

Reply:
xbuild is not something which you would like to use for .NET development, even if you use mono. I'm not that familiar with `old Mono` (which is before dotnet and Mono begin consolidating) but Mono 6 for example use `msbuild` for the build.

commented line is junk.